assets/blocks/course-outline/style.scss

```diff
@@ -0,0 +1,13 @@
+.wp-block-sensei-lms-course-outline {
+	border: solid 1px #000;
+}
+
+.wp-block-sensei-lms-module-outline {
+	background: #CCC;
+	border: solid 2px red;
```

Comment:
Not sure about the colors here, maybe something a bit more subtle :)

Reply:
Absolutely! haha 😂! Actually, I just added some temporary colors until we get the tasks to implement the correct layout.